exporters/tests/test_pelorus_webhook_handler.py

```diff
@@ -244,3 +247,162 @@ async def test_pelorus_receive_pelorus_payload_error(headers, json_payload):
         await handler._receive_pelorus_payload(json_payload_data)
     assert http_exception.value.detail == "Invalid payload."
     assert http_exception.value.status_code == http.HTTPStatus.UNPROCESSABLE_ENTITY
+
+
+# fmt: off
+@pytest.mark.parametrize(
+    "secret,expected_signature,json_payload_data",
+    [
+        (
+            b"My Secret",
+            "sha256=8b1e67669068057d751c49ca128fcf3b112dad8f26d2d00ece451c51b18351c1",
+            {"data": "value", "data2": "value2"},
+        ),
+        (
+            b"My Secret",
+            "sha256=8b1e67669068057d751c49ca128fcf3b112dad8f26d2d00ece451c51b18351c1",
+            {"data":"value","data2":"value2"}, # noqa
```

Comment:
and these comments?

Reply:
pylama will complain that the dict is improperly formatted, but that's what we want in this test.